issue.d/90openssl

```diff
@@ -12,11 +12,24 @@ nameopt=""
 if [ ! -z "$name" ]; then
    nameopt="-subj \"$name\""
 fi
+
+if [ "x$ICI_VERBOSE" = "xy" ]; then
```

Comment:
Why the `x`?  It's utterly and completely unnecessary!

Reply:
Because I'm old? =)

Feel free to improve.

Reply:
its not /completely/ unnecessary right... https://stackoverflow.com/questions/174119/why-do-shell-script-comparisons-often-use-xvar-xyes 

I choose to do stuff like this because you don't always control what manner of insanity folks will do to their shells sometimes... and the debugging is a bitch.

Reply:
Ah, the leading dash case.  Yeahok...